src/test/java/space/obminyashka/items_exchange/i18n/InternationalizationTest.java

```diff
@@ -48,15 +48,15 @@ void testDefaultLocalizationWithoutAccept_LanguageHeader() throws Exception {
     }
 
     @ParameterizedTest
-    @ValueSource(strings = {"ru", "ru-RU"})
-    void testRussianLocalizationWithAccept_LanguageHeaderIsRu(String locale) throws Exception {
+    @ValueSource(strings = {"ua", "ua-UA"})
+    void testUkrainianLocalizationWithAccept_LanguageHeaderIsRu(String locale) throws Exception {
         MvcResult mvcResult = mockMvc.perform(get(ApiKey.USER_MY_INFO)
```

Comment:
Please change the end of the test's name to isUa

Reply:
Nice catch!